rebirthdb/_dump.py

```diff
@@ -166,10 +167,13 @@ def main(argv=None, prog=None):
 
             try:
                 _export.run(options)
-            except Exception as e:
+            except Exception as exc:
+                default_logger.exception(str(exc))
+
                 if options.debug:
```

Comment:
Should this behavior be rolled into the logger?

Reply:
You mean call the `str` method? If yes, I would vote to do it, but in a separate PR to not mix things.

Reply:
I was looking at the debug option. Separate PRs for both pieces would work.